pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.6.2</version>
```

Comment:
Used folio-spring-base library of 3.0.1 has spring-boot-starter-parent 2.6.2.  folio-spring-base  version should be upgraded also.

Reply:
Changes made to use the latest version of folio-spring-base.version 3.0.3. File committed for the same.